src/win32cfg.h

```diff
@@ -92,11 +98,7 @@
 #define SIZEOF_UNSIGNED_INT 4
 
 /* The size of `unsigned long', as computed by sizeof. */
-#ifdef _WIN64
-#define SIZEOF_UNSIGNED_LONG 8
-#else
 #define SIZEOF_UNSIGNED_LONG 4
-#endif
```

Comment:
I meant the delta in general. Thanks for the link; I wonder what the prior `#ifdef` was trying to do.